doc/developers/contributing.rst

```diff
@@ -14,6 +14,26 @@ adding new algorithms, and the best way to contribute and to help the project
 is to start working on known issues.
 See :ref:`easy_issues` to get started.
 
+.. topic:: **Our community, our values**
+
+    We are a community based on openness and friendly, didactic,
+    discussions.
+
+    We aspire to treat everybody equally, and value their contributions.
+
+    Decisions are made based on technical merit and consensus.
```

Comment:
Some techies are dicks, intentionally and otherwise. I realised yesterday that a code contribution I made to a project was rewritten by the primary maintainer of that project, who praised and retained my unit tests, but didn't leave my commit or any attribution in the repository. I know this too is not really covered by the code of conduct... but being considerate and valuing contributions is not as easy as it looks!